mkdocs/themes/readthedocs/base.html

```diff
@@ -61,7 +61,7 @@
     function gtag(){dataLayer.push(arguments);}
     gtag('js', new Date());
 
-    gtag('config', '{{ config.theme.analytics.gtag }}');
+    gtag('config', '{{ config.theme.analytics.gtag }}'{%- if config.theme.analytics.anonymize_ip %}, {'anonymize_ip': true}{%- endif %});
```

Comment:
This is because MkDocs supports both the old 'isogram' analytics, as well as the newer gtag version. RTD had deprecated the 'isogram' version, but I didn't want to do that without comment.

Reply:
I see now that MkDocs has [deprecated it as well](https://github.com/mkdocs/mkdocs/commit/80feaf03a42dacdd0740259b647014d5f0f9d839). Let me know if I should just delete the isogram code.